spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -672,6 +672,11 @@ public void setDefaultTopic(String defaultTopic) {
 		 */
 		private Long ackTime;
 
+		/**
+		 * True if this should be batch listener.
+		 */
+		private Boolean batchListener;
```

Comment:
Added this change as [separate commit](https://github.com/spring-projects/spring-boot/pull/9448/commits/daaa4da46b363477aba705a31af2c17740c2a714), let me know what you think. Thanks!